microsite/data/plugins/time-saver.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: TimeSaver
+author: tduniec
+authorUrl: https://github.com/tduniec
+category: Discovery
+description: Visualise your time saved using Backstage Scaffolder templates
+documentation: https://github.com/tduniec/backstage-timesaver-plugin/blob/main/README.md
+iconUrl: img/tsLogo.png
```

Comment:
This image is still not part of the PR. Make sure to either add it in the PR or change this to an URL

Reply:
updated